src/hearings/containers/request-hearing/hearing-change-reason/hearing-change-reason.component.ts

```diff
@@ -1,32 +1,43 @@
 import {Component, OnDestroy} from '@angular/core';
-import {ACTION, HearingChangeReasonMessages } from '../../../models/hearings.enum';
+import {ACTION, HearingChangeReasonMessages, HearingSummaryEnum } from '../../../models/hearings.enum';
 import {HearingsService} from '../../../services/hearings.service';
 import * as fromHearingStore from '../../../store';
 import {RequestHearingPageFlow} from '../request-hearing.page.flow';
 import {OnInit} from '@angular/core';
 import {FormArray, FormBuilder, FormGroup} from '@angular/forms';
 import {ActivatedRoute, Router} from '@angular/router';
-import {Store} from '@ngrx/store';
+import {select, Store} from '@ngrx/store';
 import { LovRefDataModel } from '../../../models/lovRefData.model';
+import { Observable, Subscription } from 'rxjs';
 @Component({
   selector: 'exui-hearing-change-reason',
   templateUrl: './hearing-change-reason.component.html',
 })
 export class HearingChangeReasonComponent extends RequestHearingPageFlow implements OnInit, OnDestroy {
   public hearingChangeReason: LovRefDataModel[];
   public hearingChangeReasonForm: FormGroup;
-  public validationErrors: { id: string, message: string }[] = [];
+  public errors: { id: string, message: string }[] = [];
   public selectionValid: boolean = true;
+  public hearingState$: Observable<fromHearingStore.State>;
+  public sub: Subscription;
 
   constructor(protected readonly route: ActivatedRoute,
               protected readonly router: Router,
               private readonly formBuilder: FormBuilder,
               protected readonly hearingStore: Store<fromHearingStore.State>,
               protected readonly hearingsService: HearingsService) {
       super(hearingStore, hearingsService);
+      this.hearingState$ = this.hearingStore.pipe(select(fromHearingStore.getHearingsFeatureState));
```

Comment:
it could get the getHearingRequestLastError by:
```suggestion
      this.hearingRequestLastError$ = this.hearingStore.pipe(select(fromHearingStore.getHearingRequestLastError));
```
rather than load the whole hearing state.